functional-tests/tests/rpc_exec_update.py

```diff
@@ -15,9 +14,12 @@ def main(self, ctx: flexitest.RunContext):
 
         # create both btc and sequencer RPC
         seqrpc = seq.create_rpc()
-        time.sleep(2)
-
-        recent_blks = seqrpc.strata_getRecentBlockHeaders(1)
+        recent_blks = wait_until_with_value(
+            lambda: seqrpc.strata_getRecentBlockHeaders(1),
+            lambda value: value is not None,
+            error_with="Blocks not generated",
+            timeout=2,
+        )
```

Comment:
I think there should already be a fn for this?  Not sure.  But really this should be phrased as a "wait for next block" fn.

Reply:
No there wasn't. Created one.